content/docs/user-guide/settings/az-console-how-to-use-settings-registry.md

```diff
@@ -0,0 +1,173 @@
+---
+description: Describes how to execute AZ Console Commands using the Settings Registry
+title: Using AZ::Conosle to access the Settings Registry
+linktitle: Using AZ::Conosle to access the Settings Registry
+weight: 400
+---
+
+Preamble
+========
+
+The AZ::Console has been extended with support for the Settings Registry to allow invoking console commands via setting a JSON value underneath a specific key in the Settings Registry.
+The AZ::Console hooks into the Settings Registry Notification System as well as the JSON Patching Reporting system in order to detect changes to Settings Registry keys underneath the JSON object of "/Amazon/AzCore/Runtime/ConsoleCommands" and "/O3DE/Autoexec/ConsoleCommands"
+Where to place a console command depends on whether the user desires the console command should be merged to the bootstrap.game.\<config>.\<platform>.setreg file and therefore in the GameLauncher/ServerLauncher
+
+
+
+SettingsRegistry ConsoleCommand key differences
+
+`/Amazon/AzCore/Runtime/ConsoleCommands` - ConsoleCommands underneath this object is NOT merged to the bootstrap.game.\<config>.\<platform>.setreg
+
+`/O3DE/Autoexec/ConsoleCommands` - ConsoleCommands underneath this object is merged into the bootstrap.game.\<config>.\<platform>.setreg and therefore will execute in the GameLauncher/ServerLauncher
+
+
+
+Running Console Commands from a file
+====================================
+
+The AZ::Console supports running console commands from a via the AZ::IConsole::ExecuteConfigFile() function.
+That function is able to load console commands from Windows INI Style files(\*.cfg), JSON Merge Patch files which can be authored as normal json files(\*.setreg) and JSON Patch files(\*.setregpatch)
+
+Config file with Console Commands(\*.cfg)
+-----------------------------------------
+
+**user.cfg**
+
+```ini
+testInit = 3
+testInit 3
+testBool true
+testChar Q
+testUInt64 18446744073709551615
+testFloat 1.0
+testDouble 2
+testString Stable
+ConsoleSettingsRegistryFixture.testClassFunc Foo Bar Baz
+```
+
+Settings Registry Merge Patch file with Console Commands(\*.setreg)
+-------------------------------------------------------------------
+
+**user.setreg**
+
+```json
+{
+    "Amazon": {
+        "AzCore": {
+            "Runtime": {
+                "ConsoleCommands": {
+                    "testInit": 3,
+                    "testBool": true,
+                    "testChar": "Q",
+                    "testUInt64": 18446744073709551615,
+                    "testFloat": 1.0,
+                    "testDouble": 2,
+                    "testString": "Stable",
+                    "ConsoleSettingsRegistryFixture.testClassFunc": "Foo Bar Baz"
+                }
+            }
+        }
+    },
+    "O3DE": {
+        "Autoexec": {
+            "ConsoleCommands": {
+                "LoadLevel": "path/to/level.spawnable",
```

Comment:
These do not appear to be from user.cfg above, so either omit or provide some context.

Reply:
Or I can just add them to add the examples including the .cfg example

This particular section is how configuring for the game launcher and a release scenario, so that the level is automatically loaded and the launcher doesn't try to connect to the AP.